<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreListeMembreAdministration</class>
 <widget class="QWidget" name="FenetreListeMembreAdministration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FenetreListeMembreAdministration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold;
	text-align: center;
}

</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>572</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>332</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelNomMembreAdministration">
         <property name="text">
          <string>Nom

Nom1
Nom2
.
.
.
Nom n
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelPrenomMembreAdministration">
         <property name="text">
          <string>Prenom

Prenom1
Prenom2
.
.
.
Prenom n
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelPrivilegeMembreAdministration">
         <property name="text">
          <string>Contact 

Contact1
Contact2
.
.
.
Contact n
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
